src/app/utils/useMarzipano.ts

```diff
@@ -1,15 +1,21 @@
 import { useStateRef } from '@amsterdam/arm-core'
-import throttle from 'lodash.throttle'
+import debounce from 'lodash.debounce'
 import Marzipano from 'marzipano'
-import { RefObject, useEffect, useState } from 'react'
+import { RefObject, useEffect, useState, useCallback } from 'react'
 import { getOrientation } from '../../panorama/services/marzipano/marzipano'
 import { PanoParam } from '../pages/MapPage/query-params'
 
-const useMarzipano = (ref: RefObject<HTMLElement>) => {
+const useMarzipano = (
+  ref: RefObject<HTMLElement>,
+): {
+  marzipanoViewer: any
+  marzipanoViewerRef: RefObject<HTMLElement>
+  currentMarzipanoView: any
```

Comment:
According to the types defined in this method `currentMarzipanoView` is of type `PanoParam | null`. This type should be updated to reflect this.

```suggestion
  currentMarzipanoView: PanoParam | null
```